### PR TITLE
Validator --skipGeometryCheck and --validConfig options

### PR DESCRIPTION
**Pull request by domi4484**
Depends on https://github.com/opengisch/QgisModelBakerLibrary/pull/20

image

### CODE REVIEW

**Comment by signedav** [comment]
Nice. Can you make a file browser option for the config file? See existing implementation in ili2db_options.py

**Comment by domi4484** [comment]
@signedav do you mean having the possibility to set it into this form? Or just so that the path is taken/saved to settings?

File: QgisModelBaker/gui/validate.py [hunk 1]
```diff
@@ -464,3 +475,10 @@ def _get_feature_in_project(self, t_ili_tid):
                     if feature.attributes()[idx] == t_ili_tid:
                         return layer, feature
         return None, None
+
+    def _select_validator_config_file(self):
+        filename, _ = QFileDialog.getOpenFileName(
```
Comment (signedav) [inline QgisModelBaker/gui/validate.py:480]: You can use here make_file_selector from qt_utils.py doing not much but having less lines of code here.

Anyway, would be nice to pass a filter on "ini" and "INI".

**Review by signedav — APPROVED** [review]

**Comment by signedav** [comment]
> @signedav do you mean having the possibility to set it into this form? Or just so that the path is taken/saved to settings?

Ah sorry. I was confused by the tooltip in your screenshot. :-)

**Comment by signedav** [comment]
Do you plan to append the configuration part here or in another PR?

File: QgisModelBaker/gui/validate.py (hunk 1 above)
Comment (domi4484) [reply QgisModelBaker/gui/validate.py]: Thanks I changed that as suggested

**Comment by domi4484** [comment]
What do you mean with configuration part? this one? https://github.com/opengisch/QgisModelBakerLibrary/pull/20

**Comment by signedav** [comment]
No the passing of the parameter. But I've overseen it. Fine. So this is good here.

**Comment by signedav** [comment]
Merge when https://github.com/opengisch/QgisModelBakerLibrary/pull/20 is done.

**Comment by signedav** [comment]
🎉

**Pull request merged**